packages/vuetify/src/components/VStepper/VStepper.sass

```diff
@@ -89,7 +89,7 @@
     width: 24px
     transition: .3s map-get($transition, 'fast-in-fast-out')
 
-    .v-icon
+    .v-icon.v-icon
```

Comment:
```
.v-icon
  .v-icon
    font-size: 18px
```

Reply:
@MajesticPotatoe it will compile into `.v-icon .v-icon`, what has another meaning.

Reply:
gotcha, ☕️